Clarify tested dists for homebrew (#1120)
redis/docs: content/operate/oss_and_stack/install/install-redis/install-redis-on-mac-os.md

This guide shows you how to install Redis on macOS using Homebrew. Homebrew is the easiest way to install Redis on macOS. If you'd prefer to build Redis from the source files on macOS, see [Installing Redis from Source]({{< relref "/operate/oss_and_stack/install/install-redis/install-redis-from-source" >}}).

{{< note >}}

> Note that the `homebrew` Redis Community distributon is only tested on MacOS.
> 
> **The `homebrew` installation method isn't officially supported on Ubuntu / other Linux distributions.**

Reviewer: I hope you don't mind a simplification.

```suggestion
The Homebrew distribution of Redis Community Edition is only supported on macOS.
```
